src/core/space_interaction_API.jl

```diff
@@ -56,7 +56,7 @@ remove_agent_from_space!(agent, model) = notimplemented(model)
 # %% IMPLEMENT: Neighbors and stuff
 #######################################################################################
 """
-    nearby_ids(position, model::ABM, r = 1; kwargs...) → ids
+    nearby_ids(pos, model::ABM, r = 1; kwargs...) → ids
```

Comment:
Why is such a change made? Surely we can agree that `position` is more readable than `pos`. Besides, the docstring continues to refer to `position` which no longer exists.

Reply:
All the other functions use the `pos` convention

Reply:
ok, actually not all, but most

Reply:
ok we can stay with `pos` but you need to update the docstrings to also refer to `pos` and not to `position`